tdms/src/fields/electric.cpp

```diff
@@ -5,8 +5,8 @@
 using namespace std;
 
 
-void ElectricField::add_to_angular_norm(double f, int n, int Nt, SimulationParameters &params) {
-  angular_norm += phasor_norm(f, n, params.omega_an, params.dt, Nt);
+void ElectricField::add_to_angular_norm(int n, int Nt, SimulationParameters &params) {
```

Comment:
`add_to_angular_norm` seems to be the same for all sub-classes of `Field`, so makes sense to move it up to the `Field` class?

Reply:
indeed it does 👍🏼